changelog.xml

```diff
@@ -436,6 +436,13 @@
 		</rollback>
 	</changeSet>
 
+	<changeSet id="unique-constraint-plugin-endpoints" author="nicolaikrebs" dbms="mariadb">

```

Comment:
sqlite can also check this (but I don't know how to do this in liquibase; should look similar though)